include/revng/Yield/Support/PTML.h

```diff
@@ -0,0 +1,159 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <algorithm>
+#include <climits>
+
+#include "llvm/ADT/SmallString.h"
+#include "llvm/ADT/StringMap.h"
+
+#include "revng/ADT/Concepts.h"
+#include "revng/Support/Debug.h"
+#include "revng/Support/YAMLTraits.h"
+
+namespace yield::ptml {
+
+namespace tags {
+
+inline constexpr auto Div = "div";
+inline constexpr auto Span = "span";
+
+} // namespace tags
+
+namespace attributes {
+
+inline constexpr auto scope = "data-scope";
+inline constexpr auto token = "data-token";
+inline constexpr auto locationDefinition = "data-location-definition";
+inline constexpr auto locationReferences = "data-location-references";
+inline constexpr auto modelEditPath = "data-model-edit-path";
```

Comment:
This does not follow conventions for global variables casing

Reply:
Will fix, but shouldn't `clang-{format,tidy}` warn about this?